packages/amplify-category-auth/resources/auth-custom-resource/hostedUIProviderLambda.js

```diff
@@ -72,7 +72,7 @@ exports.handler = (event, context, callback) => {
           let providerList = result.Providers.map(provider => provider.ProviderName);
           let providerListInParameters = hostedUIProviderMeta.map(provider => provider.ProviderName);
           hostedUIProviderMeta.forEach(providerMetadata => {
-            if (providerList.indexOf(providerMetadata.ProviderName) > 1) {
+            if (providerList.indexOf(providerMetadata.ProviderName) > -1) {
```

Comment:
rather than index of you could use some

Reply:
it's a nit up to you if you want to change that

Reply:
this is Master branch code , I missed this when I ported these into js files